app/helpers/dashboard_helper.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module DashboardHelper
+  def complaints_created
+    current_user.complaints.created.count
+  end
+
+  def complaints_in_process
+    current_user.complaints.in_process.count
+  end
+
+  def complaints_attended
+    current_user.complaints.attended.count + current_user.complaints.attended_by_program.count
+  end
+
+  def complaints_rejected
```

Comment:
`complaints_created`, `complaints_in_process` and `complaints_rejected ` are veeeery similar
We could do something like:

```
def number_of_complaints_by_user(type:)
  current_user.complaints.where(status: type.size
end
```

Also you may find [this](https://mensfeld.pl/2014/09/activerecord-count-vs-length-vs-size-and-what-will-happen-if-you-use-it-the-way-you-shouldnt/) interesting 
TLDR: `count` always performs a SQL query, `size` will count the collection if it was previously loaded, if not it'll perform a SQL query then